src/com/google/cose/EccSigningKey.java

```diff
@@ -29,34 +29,29 @@
 import java.security.PublicKey;
 
 /** Implements COSE_Key spec for signing purposes. */
-public final class SigningKey extends Key {
+public final class EccSigningKey extends Key {
```

Comment:
Nit: Same thing as on the previous commit, this should be `Ec2SigningKey` or `EcdsaSigningKey` since `Ecc` would include Curve 25519.